tests/test_algorithm.py

```diff
@@ -1952,7 +1913,8 @@ def test_asset_date_bounds(self):
             'start_date': self.sim_params.period_start,
             'end_date': '2020-01-01',
         }])
-        with tmp_dir() as tempdir, tmp_trading_env(equities=metadata) as env:
+        with TempDirectory() as tempdir, \
```

Comment:
I think this `with` and the one above it really want to be `with temp_data_portal(...)`.